### ISSUE TITLE
It not work with dojo 1.9.1, DateTextBox, DateTimeTextBox ,etc

### ISSUE THREAD

**Issue by realjiangms** [atoha/dbootstrap#92]
First that's an amazing theme :+1: 
But I met some problem when I tried it with my dialog which contains some DateTextBox, DateTimeTextBox, etc. The dropdown button disappear and leave a small blank button.

I checked with firefox and found the root cause is, in dojo1.9.1, DropDownBox.html use 'input' instead of 'span' to present the dropdown button. That impact all widgets use template 'DropDownBox.html'. The related css element seems to be 'display: none' in .dbootstrap .dijitComboBox input.dijitArrowButtonInner and 'text-indent: -2em !important;' in .dijitTextBox input.dijitArrowButtonInner

I can workaround it by change the DropDownBox.html or hard code some hack css in my global css file.

But is it known issue or is there any graceful workaround?

**Comment by realjiangms** [comment]
The rough workaround of hacking css is:
.dbootstrap .dijitComboBox input.dijitArrowButtonInner {
    display: inline-block !important;
        text-indent: 0 !important;
        width: 1em;
}

**Comment by martinpengellyphillips** [comment]
Thanks for the report. Will take a look at this over the Christmas break.

**Comment by realjiangms** [comment]
Never mind it. I found the problem is the code in icon_support.js didn't take effect when I met this issue so the element with 'dijitArrowButtonInner' was not replaced by span. Now it works fine as I discarded the 'parseOnLoad' and moved dbootstrap line a little forward. I think we may need a wiki page as knowledge base for some comment stupid false usage.

Found your comment for this issue in icon_support.js:

To use, require this module _before_ Dijit
